scripts/environment_json.mjs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {getVersion} from "./get_version.mjs";
+import {getBuildNumber} from "./get_build_number.mjs";
+import url from "url";
+import minimist from "minimist";
+
+/*
+  Inputs:
+  => platform: the platform to generate the environment.json for
+  => flavor: the flavor of binary to build, i.e. debug or release
+
+  Outputs:
+  => the environment.json contents
+*/
+export async function environmentJson(platform, flavor) {
```

Comment:
"Flavor" is a generic name for a build variant dimension, not commonly used for debug/release.
Let's use `buildType` to align with Android or `buildMode` to align with Flutter.

Reply:
buildMode has the added benefit of overlapping with webpack, so i'll use that